src/Orchard.Web/Modules/Orchard.Layouts/Assets/JavaScript/LayoutEditor/Services/ScopeConfigurator.js

```diff
@@ -281,6 +284,10 @@
                                         element.setIsDropTarget(false);
                                         if (!!receivedElement)
                                             receivedElement.setIsFocused();
+
+                                        $scope.$root.addElement(receivedElement).done(function () {
```

Comment:
Ez miért kell ide?

Reply:
A LayoutDesignerHost.js  van egy addelement és egy editelement (a dialoggal rendelkező layout elementekre vonatkoznak ezek).
Amennyire megértettem az Angulart ez egy extension point ami nem lett implementálva nekem kell triggerelni, hogy az említettben lefusson: https://github.com/Lombiq/Orchard/pull/2/files#diff-8bd9f7dfdc3e4ceef11809bc19a054adR30

Reply:
OK, alaposan tesztelgesd végig a Layouts-ot, hogy mindezek nem rontanak-e el bármit (preview nélkül is).